src/components/AppContent/AppContent.vue

```diff
@@ -48,6 +73,31 @@ export default {
 		},
 	},
 
+	data() {
+		return {
+			paneIdentifier: `pane-sizes.${this.$parent.$options.name}`,
+			paneDefaults: {
+				aside: {
+					size: 25,
+					min: 15,
+					max: 40,
+				},
+				main: {
+					size: 75,
+					min: 60,
+					max: 85,
+				},
+			},
+		}
+	},
+
+	computed: {
+		paneSizes() {
+			const clientPaneSizes = localStorage.getItem(this.paneIdentifier)
```

Comment:
Hey, `localStorage` might not be available for every browser.
Could you use the [@nextcloud/browser-storage](https://www.npmjs.com/package/@nextcloud/browser-storage) package please? :)

Reply:
Didn't know about that! I love that it has the exact same API, done! 👌